Laravel.gitignore

```diff
@@ -1,5 +1,7 @@
 vendor/
 node_modules/
+.rocketeer/
```

Comment:
Sorry for my rudeness. This is the package https://github.com/rocketeers/rocketeer